website/chatui/Program.cs

```diff
@@ -1,8 +1,29 @@
+using Microsoft.Extensions.Options;
+using chatui.Configuration;
+
 var builder = WebApplication.CreateBuilder(args);
 
-builder.Services.AddControllersWithViews();
+builder.Services.AddOptions<ChatApiOptions>()
+    .Bind(builder.Configuration)
+    .ValidateDataAnnotations()
+    .ValidateOnStart();
```

Comment:
In the deployment story for this sample, we update one of the configuration values in Azure App Service. Will doing configuration this way be "just as responsive" as the current way?  I don't remember exactly if the current way handles a reasonably live change, requires an app restart, or whatever.  Just want to make sure that this new way is no worse than the previous way.

Reply:
good catch @ckittel, hot reload is now in place

done | address from eabf6aed717b76286632c9667bc1bb39e4a61e97